<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>addBooksDialog</class>
    <widget class="QDialog" m_name="addBooksDialog">
        <property m_name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>405</width>
                <height>300</height>
            </rect>
        </property>
        <property m_name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property m_name="minimumSize">
            <size>
                <width>400</width>
                <height>300</height>
            </size>
        </property>
        <property m_name="windowTitle">
            <string>Add Books</string>
        </property>
        <property m_name="windowIcon">
            <iconset resource="../resources/images.qrc">
                <normaloff>:/icons/books_icon.png</normaloff>:/icons/books_icon.png
            </iconset>
        </property>
        <widget class="QPushButton" m_name="buttonAdd">
            <property m_name="geometry">
                <rect>
                    <x>120</x>
                    <y>260</y>
                    <width>81</width>
                    <height>21</height>
                </rect>
            </property>
            <property m_name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property m_name="styleSheet">
                <string notr="true"/>
            </property>
            <property m_name="text">
                <string>Add</string>
            </property>
        </widget>
        <widget class="QCheckBox" m_name="checkBoxRecursive">
            <property m_name="geometry">
                <rect>
                    <x>10</x>
                    <y>120</y>
                    <width>72</width>
                    <height>19</height>
                </rect>
            </property>
            <property m_name="text">
                <string>Recursive</string>
            </property>
        </widget>
        <widget class="QLineEdit" m_name="textFolderPath">
            <property m_name="geometry">
                <rect>
                    <x>60</x>
                    <y>70</y>
                    <width>271</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" m_name="addBooksLabel">
            <property m_name="geometry">
                <rect>
                    <x>160</x>
                    <y>10</y>
                    <width>81</width>
                    <height>20</height>
                </rect>
            </property>
            <property m_name="text">
                <string>Add Books</string>
            </property>
            <property m_name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QPushButton" m_name="buttonClose">
            <property m_name="geometry">
                <rect>
                    <x>210</x>
                    <y>260</y>
                    <width>80</width>
                    <height>21</height>
                </rect>
            </property>
            <property m_name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property m_name="styleSheet">
                <string notr="true"/>
            </property>
            <property m_name="text">
                <string>Cancel</string>
            </property>
        </widget>
        <widget class="QPushButton" m_name="buttonBrowseFolders">
            <property m_name="geometry">
                <rect>
                    <x>340</x>
                    <y>70</y>
                    <width>51</width>
                    <height>21</height>
                </rect>
            </property>
            <property m_name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property m_name="styleSheet">
                <string notr="true"/>
            </property>
            <property m_name="text">
                <string>...</string>
            </property>
        </widget>
        <widget class="QProgressBar" m_name="progressBar">
            <property m_name="geometry">
                <rect>
                    <x>10</x>
                    <y>180</y>
                    <width>381</width>
                    <height>23</height>
                </rect>
            </property>
            <property m_name="value">
                <number>0</number>
            </property>
        </widget>
        <widget class="QLabel" m_name="labelFolder">
            <property m_name="geometry">
                <rect>
                    <x>10</x>
                    <y>70</y>
                    <width>41</width>
                    <height>21</height>
                </rect>
            </property>
            <property m_name="text">
                <string>Folder:</string>
            </property>
        </widget>
    </widget>
    <tabstops>
        <tabstop>textFolderPath</tabstop>
        <tabstop>buttonBrowseFolders</tabstop>
        <tabstop>checkBoxRecursive</tabstop>
        <tabstop>buttonAdd</tabstop>
        <tabstop>buttonClose</tabstop>
    </tabstops>
    <resources>
        <include location="../resources/images.qrc"/>
    </resources>
    <connections/>
</ui>
